### PR TITLE
users routes complete

### PR DESCRIPTION
**Pull request by Briahana-Mauge**

### CODE REVIEW

File: server/routes/users.js [hunk 1]
```diff
@@ -20,22 +20,76 @@ const middleWare = async (req, res, next) => {
...
+    res.json({
+      message: `There was an error!`
+    });
+  }
+}
+
+const removeUser = async (req, res) => {
+  try {
+    let id = parseInt(req.params.id);
+    await db.none(`DELETE FROM users WHERE user_id = '${id}'`);
```
Comment (joseph-p-pasaoa) [inline server/routes/users.js]: i think this is open to that SQL injection security bug that Alejo was talking about.
take the template from your lines 54-59 and something like the following instead:
let insertQuery = \`
 DELETE FROM users WHERE user_id = $1
  \`
    await db.none(insertQuery, parseInt(req.params.id));

if there's only one value i think you don't have to put it in an array form

File: server/routes/users.js [hunk 2]
```diff
@@ -20,22 +20,76 @@ const middleWare = async (req, res, next) => {
...
+  }
+}
+
+const singleUser = async (req, res) => {
+  try {
+    let id = parseInt(req.params.id);
+    console.log(id);
+    let user = await db.any(`SELECT * FROM users WHERE user_id = '${id}'`);
+    console.log(user)
+    res.json({
```
Comment (joseph-p-pasaoa) [inline server/routes/users.js:41]: remove the console logs in lines 39 and 41

File: server/routes/users.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const log = console.log;
```
Comment (DouglasMacKrell) [inline server/routes/users.js:10]: Test comment!

Comment (Briahana-Mauge) [reply server/routes/users.js:10]: i can see this

**Review by joseph-p-pasaoa — COMMENTED** [review]
@Bpmauge

**Comment by Briahana-Mauge** [comment]
@DouglasMacKrell @KathyPuma @joseph-p-pasaoa 
ready for review

**Review by joseph-p-pasaoa — APPROVED** [review]

**Review by DouglasMacKrell — APPROVED** [review]

**Pull request merged**